sphere-models/src/it/java/io/sphere/sdk/products/attributes/LocalizedToStringProductAttributeConverterTest.java

```diff
@@ -0,0 +1,222 @@
+            @Override
+            protected Collection<String> longAttributes() {
+                return singletonList(ATTR_NAME_NUMBER);
+            }
+
+            @Override
+            protected Collection<String> longSetAttributes() {
+                return singleton(ATTR_NAME_NUMBER_SET);
+            }
+        };
+    }
+
+    @AfterClass
+    public static void cleanupScenario() {
+        ProductsScenario1Fixtures.destroy(client(), data);
+    }
+
+    @Test
+    public void convertStringSet() {
+        assertThat(converting(ATTR_NAME_TEXT_SET)).isEqualTo("bar, foo");
+    }
+
+    @Ignore
+    @Test
+    public void convertProductTypeReferenceSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertProductTypeReference() {
+    }
+
+    @Test
+    public void convertProductReferenceSet() {
+        assertThat(converting(ATTR_NAME_REF)).isNull();
+    }
+
+    @Test
+    public void convertProductReference() {
+        assertThat(converting(ATTR_NAME_REF_SET)).isNull();
+    }
+
+    @Test
+    public void convertMoneySet() {
+        assertThat(converting(ATTR_NAME_MONEY_SET)).isEqualTo("500,00 EUR, 1.000,00 USD");
+    }
+
+    @Test
+    public void convertLongSet() {
+        assertThat(converting(ATTR_NAME_NUMBER_SET)).isEqualTo("5, 10");
+    }
+
+    @Test
+    public void convertLong() {
+        assertThat(converting(ATTR_NAME_NUMBER)).isEqualTo("5");
+    }
+
+    @Test
+    public void convertTimeSet() {
+        assertThat(converting(ATTR_NAME_TIME_SET)).isEqualTo("22:05, 23:06");
+    }
+
+    @Test
+    public void convertTime() {
+        assertThat(converting(ATTR_NAME_TIME)).isEqualTo("22:05");
+    }
+
+    @Test
+    public void convertLocalizedStringSet() {
+        assertThat(converting(ATTR_NAME_LOC_TEXT_SET)).isEqualTo("");
+    }
+
+    @Test
+    public void convertLocalizedEnumValueSet() {
+        assertThat(converting(ATTR_NAME_LOC_ENUM_SET)).isEqualTo("drei, zwei");
+    }
+
+    @Ignore
+    @Test
+    public void convertIntegerSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertInteger() {
+    }
+
+    @Test
+    public void convertEnumValueSet() {
+        assertThat(converting(ATTR_NAME_ENUM_SET)).isEqualTo("three, two");
+    }
+
+    @Ignore
+    @Test
+    public void convertDoubleSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertDouble() {
+    }
+
+    @Test
+    public void convertDateTimeSet() {
+        assertThat(converting(ATTR_NAME_DATE_TIME_SET)).isEqualTo("11.09.01 22:05, 12.10.02 23:06");
+    }
+
+    @Test
+    public void convertDateSet() {
+        assertThat(converting(ATTR_NAME_DATE_SET)).isEqualTo("11.09.2001, 12.10.2002");
+    }
+
+    @Ignore
+    @Test
+    public void convertChannelReferenceSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertChannelReference() {
+    }
+
+    @Ignore
+    @Test
+    public void convertCategoryReferenceSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertCategoryReference() {
```

Comment:
From a category, for example, as we know it's a category and the fields categories have, by default I would show the `ref.obj.name` + locale if the reference is expanded and the `ref.id` if it's not. Just a nice-to-have, of course. Mostly asking if it's possible.